io/key/key.go

```diff
@@ -64,6 +65,23 @@ type EditEvent struct {
 	Text string
 }
 
+// VirtualKeyboardMode changes the on-screen-keyboard type. That hints the
+// type of data that might be entered by the user.
+type KeyboardMode uint8
```

Comment:
I don't know if the name is right. Maybe `KeyboardHint` or `KeyboardFlag`? 😶

Reply:
How about InputMode?